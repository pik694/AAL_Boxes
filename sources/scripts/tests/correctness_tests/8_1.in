8
40 23 84
49 86 61
56 92 76
10 8 73
56 62 16
89 1 32
22 74 46
77 67 35

