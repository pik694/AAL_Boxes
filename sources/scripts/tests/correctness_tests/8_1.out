5 3 6 7 4 0 1 2 
1045579->630517
